packages/imready/src/reactive.ts

```diff
@@ -0,0 +1,153 @@
+import { Observer, observe, reactive, ReactiveAdapter, ReactiveObject } from "@cfcs/core";
+import { EVENTS } from "./consts";
+import ImReady from "./ImReady";
+import {
+  ImReadyEvents,
+  ImReadyHooksProps,
+  ImReadyReactiveState,
+} from "./types";
+import { toArray } from "./utils";
+
+export interface ImReadyData {
+  props: Partial<ImReadyHooksProps>;
+}
+
+export type ReactiveImReady = ReactiveObject<{
+  imReady: ImReady;
+  children: HTMLElement[];
+  preReadyCount: number;
+  readyCount: number;
+  errorCount: number;
+  totalErrorCount: number;
+  totalCount: number;
+  isPreReady: boolean;
+  isReady: boolean;
+  hasError: boolean;
+  isPreReadyOver: boolean;
+  register(ref?: any): any;
```

Comment:
change the type string | Ref | with HTMLElement etc.

Reply:
I checked the existing ImReady interface of each framework and there is no case about passing a string to `register`.
If my understanding is correct, can I define this type as `ref?: HTMLElement | Ref<HTMLElement>`?